spec.bs

```diff
@@ -1165,6 +1369,173 @@ Note that there is no `paymentRequestOrigin` field in
 {{CollectedClientAdditionalPaymentData}}, because the origin of the calling
 frame is already included in {{CollectedClientData}} of [[webauthn-3]].
 
+# Browser Bound Key Store - <dfn interface>BrowserBoundKeyStore</dfn> Internal Component # {#sctn-browser-bound-key-store}
+
+This component in the user agent manages platform-dependant cryptographic
+keypairs including their association to SPC credentials (i.e. passkeys). This
+section explains the |BrowserBoundKeyStore| and procedures for creating,
+binding, retrieving, and signing using the browser bound keypairs.
+
+The browser bound key store contains |browser_bound_map|, a map from
+[[=Credential ID=]] to a keypair identifier. The user agent stores
+this map on disk.
+
+## Creating a KeyPair Internal Method ## {#sctn-creating-a-keypair}
+
+<dfn for="BrowserBoundKeyStore" method>CreateKeyPair(allowed_algorithms)</dfn>:
+
+Given |allowed_algorithms| a {{PublicKeyCredentialParameters}} object from
+{{AuthenticationExtensionsPaymentInputs/browserBoundPubKeyCredParams}}, creates a
+keypair unless no allowed algorithms match, and returns a tuple of |bbk|, the
+keypair handle, and |bbk_id|, the keypair identifier along with a
+keypair identifier.
+
+1. If the |allowed_algorithms| are an empty list, set this list to a
+    default list containing the following in order:
+    * {{PublicKeyCredentialParameters/type}} = {{PublicKeyCredentialType}} with {{PublicKeyCredentialParameters/alg}} = -7 ("ES256").
+    * {{PublicKeyCredentialParameters/type}} = {{PublicKeyCredentialType}} with {{PublicKeyCredentialParameters/alg}} = -257 ("RS256").
+
+1. Remove any entries from |allowed_algorithms| that are not of type
+    {{PublicKeyCredential}}.
+
+    Issue: <span style="background-color: red">TODO</span>: Should we expand
```

Comment:
I think we should keep this for now. SPC has no business talking about non-PublicKeyCredentials at this point in time.